src/Fixer/SpecifyArgSeparatorFixer.php

```diff
@@ -0,0 +1,153 @@
+    {
+        return $tokens->isTokenKindFound(T_STRING);
+    }
+
+    public function isRisky(): bool
+    {
+        return true;
+    }
+
+    public function fix(\SplFileInfo $file, Tokens $tokens): void
+    {
+        foreach ($tokens as $index => $token) {
+            if ($token->isGivenKind(T_STRING) && $token->getContent() === 'http_build_query') {
+                if ($this->isNotFunctionCall($tokens, $index)) {
+                    continue;
+                }
+
+                $this->fixFunction($tokens, $index);
+            }
+        }
+    }
+
+    public function getName(): string
+    {
+        return self::class;
+    }
+
+    public function getPriority(): int
+    {
+        // Should be run after SingleQuoteFixer (priority 0) and ArraySyntaxFixer (priority -1)
+        return -1;
+    }
+
+    public function supports(\SplFileInfo $file): bool
+    {
+        return true;
+    }
+
+    private function fixFunction(Tokens $tokens, int $functionIndex): void
+    {
+        $openParenthesisIndex = $tokens->getNextTokenOfKind($functionIndex, ['(']);
+        if ($openParenthesisIndex === null) {
+            return;
+        }
+
+        $closeParenthesisIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $openParenthesisIndex);
+
+        $argumentCount = (new ArgumentsAnalyzer())
+            ->countArguments($tokens, $openParenthesisIndex, $closeParenthesisIndex);
+
+        // When third argument is already present, make sure it is not null or override its value
+        if ($argumentCount >= 3) {
+            $thirdArgumentType = $this->getThirdArgumentInfo($tokens, $openParenthesisIndex, $closeParenthesisIndex);
+
+            if ($thirdArgumentType === null) {
+                return;
+            }
+
+            if ($thirdArgumentType->getName() === 'null') {
+                $this->setArgumentValueToAmp($tokens, $thirdArgumentType->getStartIndex());
+            }
+
+            return;
+        }
+
+        $tokensToInsert = [];
+
+        // Add second argument if not defined
+        if ($argumentCount === 1) {
+            $tokensToInsert[] = new Token(',');
+            $tokensToInsert[] = new Token([T_WHITESPACE, ' ']);
+            $tokensToInsert[] = new Token([T_STRING, 'null']);
+        }
+
+        // Add third argument (arg separator): ", &"
+        if ($argumentCount < 3) {
+            $tokensToInsert[] = new Token(',');
+            $tokensToInsert[] = new Token([T_WHITESPACE, ' ']);
+            $tokensToInsert[] = new Token([T_STRING, "'&'"]);
+        }
+
+        if (!empty($tokensToInsert)) {
+            $beforeCloseParenthesisIndex = $tokens->getPrevNonWhitespace($closeParenthesisIndex);
+            $tokens->insertAt($beforeCloseParenthesisIndex + 1, $tokensToInsert);
+        }
+    }
+
+    /**
+     * Return false if this is function import or function definition (and not function call)
+     */
+    private function isNotFunctionCall(Tokens $tokens, int $index): bool
```

Comment:
Neither am I, but this function in fact doesn't prove it is function call when you just negate the logic - there are IMHO many edge cases. What it can is just to detect the main scenarios when it is not a function call :). So from my POV, the other name would be inaccurate and misleading :).
🤷

Reply:
Well.. 🤔 
Ok in this case I think it might be ok..

Reply:
@MortalFlesh I thought about it again and changed it in the end... The objection still applies, but maybe its less harm like this :man_shrugging: .